apps/omg_watcher/lib/omg_watcher/block_getter.ex

```diff
@@ -350,4 +350,12 @@ defmodule OMG.Watcher.BlockGetter do
   defp check_in_to_coordinator(synced_height), do: RootChainCoordinator.check_in(synced_height, :block_getter)
 
   defp update_status(%Core{} = state), do: Status.update(Core.chain_ok(state))
+
+  defp publish_data([%{event_signature: event_signature} | _] = data) do
+    # String.split("BlockSubmitted(uint256)", "(")
```

Comment:
Maybe worthwhile to move this out of the method and a more formal comment on this line? While I think I understood the meaning behind this it looks like commented code that could be deleted.